analyzers/tests/SonarAnalyzer.UnitTest/TestCases/SymbolicExecution/Roslyn/HashesShouldHaveUnpredictableSalt.cs

```diff
@@ -14,40 +14,40 @@ class Program
     public void ShortAndConstantSaltIsNotCompliant()
     {
         var shortAndConstantSalt = new byte[15];
-        var pdb1 = new PasswordDeriveBytes(passwordBytes, shortAndConstantSalt);                                                    // FIXME Non-compliant {{Make this salt unpredictable.}}
+        var pdb1 = new PasswordDeriveBytes(passwordBytes, shortAndConstantSalt);                                                    // Noncompliant {{Make this salt unpredictable.}}
```

Comment:
I cannot leave a comment in that line but also here line 1 can be removed. System is an unused namespace.

Reply:
It's needed for the type `System.Random` in the [SystemRandomIsNotCompliant method](https://github.com/SonarSource/sonar-dotnet/blob/ed9df5fedaf9a2f04f9361f486d67b187a202d7f/analyzers/tests/SonarAnalyzer.UnitTest/TestCases/SymbolicExecution/Roslyn/HashesShouldHaveUnpredictableSalt.cs#L100).